### DISCUSSION TITLE
이번 주차 참여는 힘들 것 같습니다.

### DISCUSSION THREAD

**Discussion by evelon** [ThinkAboutSoftware/AcademicConference#193] [General]
책을 제대로 읽지 않아서 이번 주는 참여하지 못할 것 같습니다. 죄송합니다. 😢

**Comment by jongfeel** [comment]
그렇군요 아쉽게 됐네요.

다음 챕터는 분량도 많고 내용도 조금 어렵다 보니
미리 읽고 준비하시면 좋을 것 같습니다.

저는 미리 지난 주 부터 챕터 2를 읽고 있는데요
정리해야 할 양이 많더라고요.
